### DISCUSSION TITLE
xmake怎么才能嵌套调用？

### DISCUSSION THREAD

**Discussion by Domain** [xmake-io/xmake#5618] [Q&A] [answered]
我也想实现这个https://github.com/xmake-io/xmake/issues/2969#issuecomment-2348321648
也参考了这个https://github.com/xmake-io/xmake/issues/612#issuecomment-552171062
我试过了os.run("xmake build --all")，这个会卡死；
task.run("build", { all = true })，这个会提示：error: co_group(build): already exists!
process.open("xmake build --all")，这个倒是没有报错，但是什么都没干。

**Comment by waruqi** [comment, marked as answer]
不支持，有锁 。否则会改坏 cache 。

**Reply by Domain** [reply]
好吧，看来只能用其他方法实现了
